classes/class-bsf-custom-fonts-render.php

```diff
@@ -132,9 +132,14 @@ public function __construct() {
 			add_filter( 'fl_builder_font_families_system', array( $this, 'bb_custom_fonts' ) );
 
 			// Add font files style.
-			add_action( 'wp_enqueue_scripts', array( $this, 'preload_styles' ), 1 );
-			add_action( 'init', array( $this, 'add_block_assets_style' ) );
+			// Add font files style.
+			if ( is_plugin_active( 'thrive-apprentice/thrive-apprentice.php' ) || is_plugin_active( 'thrive-product-manager/thrive-product-manager.php' ) ) {
```

Comment:
Can we use `defined()` condition here, instead of is_plugin_active() ?

Reply:
Resolved in this PR - https://github.com/brainstormforce/custom-fonts/pull/113